packages/odyssey-react/src/components/Box/Box.tsx

```diff
@@ -296,6 +298,31 @@ export interface BoxProps {
    * Content selection
    */
   userSelect?: "none" | "text" | "contain" | "all" | "auto";
+  /**
+   * Whether to apply the body text color class
+   * @default body
+   */
+  color?: "body" | false;
+  /**
+   * Whether to apply the normal font weight class
+   * @default regular
+   */
+  fontWeight?: "regular" | false;
+  /**
+   * Whether to apply the normal font style class
+   * @default normal
+   */
+  fontStyle?: "normal" | false;
+  /**
+   * Whether to apply the base font size class
+   * @default base
+   */
+  fontSize?: "base" | false;
+  /**
+   * Whether to apply the base line height class
+   * @default normal
+   */
+  lineHeight?: "normal" | false;
```

Comment:
Let's be consistent with what we do in the [Text component](https://github.com/okta/odyssey/pull/1201/files#diff-692cbd9f52167724305067d25015fa18c9298f6b0a644f94882a92cd6461b415R117) for line-height

Reply:
@arnoldsandoval-okta the API here is purposefully limited so that we can create "default" styles that are inherited in the DOM. We won't recreate the entire set of values exposed in Text because text should be the place folks reach for when they want an advanced text component to do fancy styling.